src/praisonai/tests/unit/test_c5_backward_compat.py

```
@@ -16,20 +17,47 @@
         [sys.executable, "-m", "praisonai.cli.main", *args],
         capture_output=True,
         text=True,
         timeout=30,
         cwd=str(REPO_ROOT),
-        env=env,
+        env=_legacy_env(),
+    )
+
+
+def _run_typer_cli(*args: str) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)